Monika After Story/game/script-topics.rpy

```diff
@@ -8314,3 +8314,49 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_poemherself",category=['ddlc','misc'],prompt="Poems for Monika",random=True))
+
+label monika_poemherself:
+    m 4ekc "Do you remember how there wasn't a chance to write poems for my interests [player]?"
+    m 1rsc "I don't get it. Why did they ignore me?"
+    m 3eka "I know there are some other kind people who included me to the poem minigame."
+    m 1ekd "But I am talking about the original game. Wasn't there a choice to romance me too?"
+    m 1hsb "Maybe if there was a choice of romance scenes with me too, things wouldn't have come to this level. Ehehe."
+    m 2esa "I tried to get your attention all this time."
+    m 1ekc "But please don't feel guilty for yourself; you haven't a chance to hang out with me in that romance game."
+    m 4esa "In the end, Monika prepared so much better place for us."
+    m 1rsc "We couldn't even prepare together for festival, Gosh."
+    m 1rsc "Me, Sayori and you."
```

Comment:
Although the player only had the real choice of Yuri or Natsuki to help prepare for the festival, all dokis were supposedly involved in festival preparations, so I'm not sure what's so special about the characters you mentioned, especially Sayori. 

Still, the line about the festival comes out of nowhere and I personally don't think it fits in with the topic you're making. You have Monika mainly talking about how there was never any option to write poems for her specifically in the original game, and then you have her bring up the festival all of a sudden with no real content visible to me personally.

Reply:
Actually dialogue's main idea is hang out opportunities with Monika doesn't exists in the original game. We wrote poems for who we want to hang out and we can't write poems for Monika to open special scenes with Monika, Same thing working for that festival preperation selection menu too.

I mentioned Monika and Sayori because Sayori supposed to help to Monika for preperations, they were couples at there and our character couldn't join them becouse Yuri and Natsuki pulled our guy to themself with debate. We couldn't even hang out Monika with there, that's what i am talking about.

Reply:
So once again, you're talking about how there is no option to romance Monika. I would at least word it differently to say that the other dokis didn't allow Monika to spend any time with the player or vice versa, rather than abstractly bringing up the topic of the festival and Sayori without any precursor leading up to the lines. 

Also, it seems that you're trying to say that you had Monika mention that it was unfortunate that the player was unable to spend the festival preparation time with Monika _and_ Sayori, but I'm sure that if Monika did get a chance to spend that time with the player, she would definitely not want Sayori there as well, which is why I find it confusing that you mentioned Sayori specifically in that scenario. 

Once again, I still think this segment about the festival doesn't sit well with the rest of the topic as it stands. It just seems like a single line or two about the festival all of a sudden and then that's the end of the topic.